composer.json

```diff
@@ -40,15 +40,20 @@
             "@stan",
             "@tests"
         ],
-        "check-cs": "php-cs-fixer fix --dry-run --diff",
-        "fix-cs": "php-cs-fixer fix",
-        "stan": "phpstan analyse -l max -c ./phpstan.neon ./src ./tests",
-        "tests": "phpunit"
+        "check-cs": "@php php-cs-fixer fix --dry-run --diff",
+        "fix-cs": "@php php-cs-fixer fix",
+        "stan": "@php phpstan analyse -l max -c ./phpstan.neon ./src ./tests",
+        "tests": "@php phpunit"
```

Comment:
:thinking: something was changed in GH actions?

Reply:
Nothing to do with github actions. This is for environments, like mine, where multiple versions of php are installed. Running composer script with other version (binary) of PHP would result in the target being run with default php binary anyway. For example `$ php74 composer tests` would run the phpunit with `php` binary anyway, which in my case is php 8.1. 🤷 

tl;dr;

This just ensures script targets are run by the same php binary that was used for composer.